.github/workflows/prow-github-actions.yaml

```diff
@@ -0,0 +1,28 @@
+name: "Prow github actions"
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  execute:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: jpmcb/prow-github-actions@v1.1.3
+        with:
+          prow-commands: '/assign 
+            /unassign 
```

Comment:
nit: I'd prefer them to alphabetically sorted if possible :)

Reply:
Yup, updated.